doc/sphinx/index.rst

```diff
@@ -15,8 +15,6 @@ Welcome to the user documentation of ESPResSo++
    tutorial.rst
    classdescription.rst
 
-   doxygen 
-
```

Comment:
Are we not integrating that anymore?

Reply:
I removed it, since doxygen comments are rarely used in the code base and and I do not see much use in a plain listing of functions without explanations. Any decent IDE can give you the same. Doxygen is still built, so if you need it I can reenable it.

Reply:
ok